tests/drum/test_fit_inference.py

```diff
@@ -0,0 +1,480 @@
+import glob
+import json
+import os
+import re
+from tempfile import NamedTemporaryFile
+
+from uuid import uuid4
+
+import numpy as np
+import pandas as pd
+import pytest
+import requests
+
+from .constants import TESTS_ROOT_PATH
+from .utils import (
+    _exec_shell_cmd,
+    _cmd_add_class_labels,
+    _create_custom_model_dir,
+)
+from .drum_server_utils import DrumServerRun
+
+from datarobot_drum.drum.common import (
+    ArgumentsOptions,
+    CUSTOM_FILE_NAME,
+    CustomHooks,
+    PythonArtifacts,
+    RunMode,
+)
+from datarobot_drum.drum.runtime import DrumRuntime
+from datarobot_drum.drum.utils import CMRunnerUtils
+
+from .constants import (
```

Comment:
nit instead of importing all these things, you could just `import .constants as constants` and `constants.TRAINING` or whatever

Reply:
then again, these are used all over the place, so ignore me

Reply:
I just don't want to change every place where this constant is used.  
And lines, where constants are used will be very long and ugly
```
 (constants.RDS, constants.REGRESSION, constants.R, None),
```

Reply:
+1